launch/launch/frontend/parser.py

```diff
@@ -74,18 +75,24 @@ def load_launch_extensions(cls):
         if cls.extensions_loaded is False:
             for entry_point in importlib_metadata.entry_points().get(
                     'launch.frontend.launch_extension', []):
-                entry_point.load()
+                try:
+                    entry_point.load()
+                except Exception:
+                    warnings.warn(f'Failed to load the launch extension: {entry_point.name}\n'
```

Comment:
@sloretz nit: it may be worth clarifying which type of launch extension we're dealing with here.

Reply:
I think the entry_point name and the stack trace from the exception in the log message should show the type dealt with. I'll merge this as is for now, but I can open up a follow up if there's something more beneficial we could add.